components/properties/src/script.rs

```diff
@@ -140,6 +148,58 @@ impl<'data> ScriptExtensions<'data> {
             ZeroSlice::from_ule_slice(script_ule_slice)
         }
     }
+
+    /// Returns whether `script` satisfies either of: 1) matches the Script
+    /// property value; 2) is contained in the Script_Extensions property value
```

Comment:
It should only return the Script property value if the code_point does not have Script_Extensions. I can't tell if the code does that.

For example, if the code_point has sc=Common and scx includes Beng, Deva, ..., Taml, ... as for U+0964 Danda, then has_script(Danda, Common) should be false.

See also https://unicode-org.github.io/icu-docs/apidoc/dev/icu4j/com/ibm/icu/lang/UScript.html#hasScript-int-int-